v0.7.0/get-started/bridge.mdx

```diff
@@ -6,6 +6,15 @@ icon: "bridge"
 
 <p> The mev-commit chain uses the native ETH tokens that need to be bridged from L1. We use a lock and mint bridge. A user CLI application can be downloaded from the [releases page](https://github.com/primev/mev-commit/releases/latest). </p>
 
+Or download the 0.7.0 bridge CLI directly from the links below:
```

Comment:
We will be doing a patch release before the next mainnet release. Possibly v0.7.1. Do we need to update this?

Reply:
Yea we should likely just update this bullet point list once the release actually happens. This was a request from murat that we make it easier for people to download the binary directly with one click